handlers/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Restart systemd
```

Comment:
Restart mattermost-hubot service
(I think systemd per se will still run after this task.)

Reply:
fixed